runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -565,7 +566,7 @@ public XWalkNavigationHistoryInternal getNavigationHistory() {
      * @param name the name injected in JavaScript.
      * @since 1.0
      */
-    @XWalkAPI
+    @XWalkAPI(reservable = true)
     public void addJavascriptInterface(Object object, String name) {
```

Comment:
Should `reservable = true` be added here? It could be added in @xzhan96 's https://github.com/crosswalk-project/crosswalk/pull/3692